prompting/tasks/task_registry.py

```diff
@@ -57,16 +59,16 @@ class TaskRegistry(BaseModel):
             datasets=[WikiDataset],
             reward_model=MultiChoiceRewardConfig,
         ),
-        TaskConfig(
-            task=ProgrammingTask,
-            probability=0.1,
-            datasets=[HuggingFaceGithubDataset],
-            reward_model=ProgrammingRewardConfig,
-        ),
+        # TaskConfig(
+        #     task=ProgrammingTask,
+        #     probability=0.1,
+        #     datasets=[HuggingFaceGithubDataset],
+        #     reward_model=ProgrammingRewardConfig,
+        # ),
         TaskConfig(
             task=WebRetrievalTask,
             # TODO: Increase probability after v2.9.0.
-            probability=0.03,
+            probability=0.13,
```

Comment:
We should fix DDGS rate-limits first, currently increasing probability doesn't make sense
```suggestion
            probability=0.13,
```